arcgis-ios-sdk-samples/Maps/Display device location/DisplayLocationViewController.swift

```diff
@@ -69,28 +69,28 @@ class DisplayLocationViewController: UIViewController, CustomContextSheetDelegat
     
     //for selection on the context sheet
     //update the autoPanMode based on the selection
-    func customContextSheet(customContextSheet: CustomContextSheet, didSelectItemAtIndex index: Int) {
+    func customContextSheet(_ customContextSheet: CustomContextSheet, didSelectItemAtIndex index: Int) {
         switch index {
         case 0:
             self.mapView.locationDisplay.stop()
         case 1:
-            self.startLocationDisplay(AGSLocationDisplayAutoPanMode.Off)
+            self.startLocationDisplay(AGSLocationDisplayAutoPanMode.off)
         case 2:
-            self.startLocationDisplay(AGSLocationDisplayAutoPanMode.Recenter)
+            self.startLocationDisplay(AGSLocationDisplayAutoPanMode.recenter)
         case 3:
-            self.startLocationDisplay(AGSLocationDisplayAutoPanMode.Navigation)
+            self.startLocationDisplay(AGSLocationDisplayAutoPanMode.navigation)
         default:
-            self.startLocationDisplay(AGSLocationDisplayAutoPanMode.CompassNavigation)
+            self.startLocationDisplay(AGSLocationDisplayAutoPanMode.compassNavigation)
         }
     }
     
     //to start location display, the first time
     //dont forget to add the location request field in the info.plist file
-    func startLocationDisplay(autoPanMode:AGSLocationDisplayAutoPanMode) {
+    func startLocationDisplay(_ autoPanMode:AGSLocationDisplayAutoPanMode) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage